src/core/stores/VNode.ts

```diff
@@ -78,6 +78,12 @@ export class VNode {
     render<T>(to = 'html'): T {
         return this.renderingEngines[to].render(this) as T;
     }
+    /**
+     * Return a new VNode that is a duplicate of this node.
+     */
+    duplicate(): VNode {
+        return new VNode(this.type, this.originalTag, this.value, this.format);
```

Comment:
Wanna call it `shallowCopy` or something?

Reply:
Shall we implement a `split` method on `VNode` ? We used to have it while programming on the whiteboard. Maybe it's time to implement the `VNodeSpec` I was talking about months ago. It would be strongly dependent on the PR where you transform VNode types into different classes though so maybe we don't want to wait for it ? If we don't want to wait for it, we can keep this duplicate method, and add a TODO where it is called:
`TODO: replace this `duplicate` method by `split` on VNode`